code/modules/interface/multiContext/context_actions.dm

```diff
@@ -697,6 +697,22 @@
 			var/obj/machinery/vehicle/V = target
 			V.use_external_speaker()
 
+	change_thruster_direction
+		name = "Lateral Thruster Direction"
+		desc = "Change the lateral thrusters to move the ship left"
+		icon_state = "thrusters_left"
+
+		execute(atom/target, mob/user)
+			..()
+			var/obj/machinery/vehicle/V = target
```

Comment:
this should have a `checkRequirements()` override to only show if the system is installed, no?

Reply:
It's already on the base type I think